scanpipe/templates/scanpipe/package_list.html

```diff
@@ -27,7 +27,7 @@
         <tbody>
           {% for package in object_list %}
             <tr class="break-word">
-              <td style="min-width: 500px;" title="{{ package.package_uid }}">
+              <td style="min-width: 500px;" title="{{ package.package_url }}">
```

Comment:
Not sure why we would want to `package_url` as title since we already display it a the text of the <a>

Reply:
I've reverted this change.